.github/workflows/upload_alerts.yml

```diff
@@ -0,0 +1,38 @@
+# upload alerts every 15 minutes
+
+name: Upload Alerts to AWS/Rockset
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+jobs:
+  upload-alerts:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Python Packages
+        run: |
+          pip3 install rockset==1.0.3
+          pip3 install boto3==1.19.12
+
+      - name: Create alerts
+        run: |
+          output=$("python tools/alerts/create_alerts.py")
+          echo "uploading following alerts"
+          echo "$output"
+          echo "::set-output name=script_output::$output"
+        id: alert_creation_step
+      - name: Upload alerts
+        uses: pytorch/test-infra/.github/actions/upload-alerts@main
+        with:
+          alerts: ${{ steps.alert_creation_step.outputs.script_output }}
+          organization: "pytorch"
```

Comment:
Just FYI: According to https://docs.github.com/en/actions/learn-github-actions/contexts#github-context, there is `GITHUB_REPOSITORY_OWNER`, i.e. `pytorch`, and `GITHUB_REPOSITORY`, i.e. `pytorch/pytorch` that we can use.  But this looks fine